pkg/ctl/topic/set_offload_policies.go

```diff
@@ -0,0 +1,205 @@
+		"AWS Credential Id to use when using driver S3 or aws-s3")
+	vc.Command.Flags().StringVarP(&flags.awsSecret, "aws-secret", "", "",
+		"AWS Credential Secret to use when using driver S3 or aws-s3")
+	vc.Command.Flags().StringVarP(&flags.s3Role, "s3-role", "", "",
+		"S3 Role used for STSAssumeRoleSessionCredentialsProvider")
+	vc.Command.Flags().StringVarP(&flags.s3RoleSessionName, "s3-role-session-name", "", "",
+		"S3 role session name used for STSAssumeRoleSessionCredentialsProvider")
+
+	vc.Command.Flags().StringVarP(&flags.fileSystemProfilePath, "file-system-profile-path", "", "",
+		"file system profile path to use when using driver filesystem")
+	vc.Command.Flags().StringVarP(&flags.fileSystemURI, "file-system-uri", "", "",
+		"file system url to use when using driver filesystem, the default to "+DefaultFileSystemURI)
+
+	vc.Command.Flags().Int64VarP(&flags.maxBlockSizeInBytes, "max-block-size-in-bytes", "",
+		DefaultMaxBlockSizeInBytes,
+		"ManagedLedger offload max block Size in bytes, s3 and google-cloud-storage requires this parameter")
+	vc.Command.Flags().Int64VarP(&flags.readBufferSizeInBytes, "read-buffer-size-in-bytes", "",
+		DefaultReadBufferSizeInBytes,
+		"ManagedLedger offload read buffer size in bytes, s3 and google-cloud-storage requires this parameter")
+	vc.Command.Flags().Int64VarP(&flags.offloadThresholdInBytes, "offload-threshold-in-bytes", "", 0,
+		"ManagedLedger offload threshold in bytes")
+	vc.Command.Flags().Int64VarP(&flags.offloadDeletionLagInMillis, "offload-deletion-lag-in-millis", "", 0,
+		"ManagedLedger offload deletion lag in bytes")
+	vc.Command.Flags().StringVarP(&flags.offloadedReadPriority, "offloaded-read-priority", "",
+		utils.TieredStorageFirst.String(),
+		"Read priority for offloaded messages. By default, once messages are offloaded to long-term storage, "+
+			"brokers read messages from long-term storage, "+
+			"but messages can still exist in BookKeeper for a period depends on your configuration. "+
+			"For messages that exist in both long-term storage and BookKeeper, "+
+			"you can set where to read messages from with the option `tiered-storage-first` or `bookkeeper-first`")
+
+	_ = vc.Command.MarkFlagRequired("driver")
+	_ = vc.Command.MarkFlagRequired("offload-threshold-in-bytes")
+
+	vc.SetRunFuncWithNameArg(func() error {
+		return doSetOffloadPolicies(vc, flags)
+	}, "the topic name is not specified or the topic name is specified more than one")
+}
+
+func doSetOffloadPolicies(vc *cmdutils.VerbCmd, flags setOffloadPoliciesFlags) error {
+	topic := vc.NameArg
+	admin := cmdutils.NewPulsarClient()
+	topicName, err := utils.GetTopicName(topic)
+	if err != nil {
+		return err
+	}
+
+	data := utils.OffloadPolicies{
+		ManagedLedgerOffloadMaxThreads:            DefaultOffloadMaxThreads,
+		OffloadersDirectory:                       DefaultOffloaderDirectory,
+		ManagedLedgerOffloadPrefetchRounds:        DefaultOffloadMaxPrefetchRounds,
```

Comment:
Are they can not be configured on the client side?

Reply:
The same implement with `pulsar-admin`,  we can support configured on the pulsar.